imgurdownloader/imgurdownloader.py

```diff
@@ -135,29 +138,66 @@ def __init__(self, imgur_url, dir_download=os.getcwd(), file_name='',
         if self.debug:
             print('album_title: ' + self.album_title)  # debug
 
-        # get section from html that contains image ID(s) and file extensions of each ID
+        image_ids = self._init_image_ids_with_regex(html=html)
+        if image_ids:
+            self.imageIDs = image_ids
+
+        json_image_ids = list(self._init_image_ids_with_json(html=html))
+        if json_image_ids:
+            self.json_imageIDs = json_image_ids
+
+        if self.debug:
+            print("imageIDs count: %s" % str(len(self.imageIDs)))  # debug
+            print("imageIDs:\n%s" % str(self.imageIDs))  # debug
+
+        self.cnt = Counter()
+        for i in self.imageIDs:
+            self.cnt[i[1]] += 1
+
+    def _init_image_ids_with_json(self, html):
+        """get section from html that contains image ID(s) and file extensions of each ID with json."""
+        """Format of the search variable.
+        item: <java dict>\n};
+        """
+        image_ids = None
+        search_failed = False
+        search = re.search('(item:.*?};)', html, flags=re.DOTALL)
+        if search:
+            try:
+                search = search.group().replace('\n', '', ).split(':', 1)[1].rsplit('}', 1)[0]
+                json_search = json.loads(search)
+                img_dicts = json_search['album_images']['images']
```

Comment:
it would fail but json search will still a valid json

here is what i got from json search 

```json
{'adConfig': {'safeFlags': ['onsfw_mod_safe', 'share', 'page_load'], 'showsAds': True, 'unsafeFlags': []},
 'animated': False,
 'bandwidth': 0,
 'date': '9 months ago',
 'datetime': '2016-09-15 00:12:30',
 'deletehash': None,
 'description': '',
 'ext': '.png',
 'favorited': False,
 'hash': 'FVRUGe2',
 'height': '1200',
 'isAd': False,
 'is_moderated': False,
 'is_safe': True,
 'is_viral': 0,
 'looping': False,
 'name': None,
 'prefer_video': False,
 'size': 2085906,
 'source': '',
 'title': 'Lock and Load [Stella Jogakuin Koukouka C3-bu]',
 'video_host': None,
 'video_source': None,
 'views': '2487',
 'width': '835'}
```
i will fix it to include single image format.